docs/lib/datastore/fragments/js/data-access/update-snippet.md

```diff
@@ -5,9 +5,9 @@ await DataStore.save(
   Post.copyOf(original, updated => {
```

Comment:
I kind of feel like this example is a little convoluted for a getting started. Why not just change a single field and leave out the status logic / update?

Reply:
Good call, and I realized that the JS snippet is the only one doing that. iOS just updates the title